src/WebUI/Common/KubeSummary.scss

```diff
@@ -58,4 +58,8 @@
             }
         }
     }
+
+    .k8s-details-header{
+        padding-bottom: 10px;
```

Comment:
header has 16px and i have added 8px, so that top will be 24px and botton be 20px.

.service-full-details-table {
    padding-top: 8px;
    padding-bottom: 20px;
    width: 100%;
    overflow-x: auto;
}

this is covering for the same.